src/main/java/com/org/jenkins/custom/jenkins/distribution/service/PluginController.java

```diff
@@ -0,0 +1,37 @@
+package com.org.jenkins.custom.jenkins.distribution.service;
+
+import com.org.jenkins.custom.jenkins.distribution.service.services.UpdateCenterService;
+import java.util.logging.Logger;
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@CrossOrigin("*")
+@RequestMapping("api/plugin")
+public class PluginController {
+
+    private final static Logger LOGGER = Logger.getLogger(PluginController.class.getName());
+    private final UpdateCenterService updateCenterService;
+
+    @Autowired
+    public PluginController(UpdateCenterService updateCenterService) {
+        this.updateCenterService = updateCenterService;
+    }
+
```

Comment:
Should I put its usage here  ? like a sort of api docs

Reply:
Yes!  Always include extra documentation, it's never bad to leave notes for people (and even yourself!) who will be looking at this later.